Makefile

```diff
@@ -1,14 +1,9 @@
-VERSION="v0.8+"
```

Comment:
any chance we can keep these in the toplevel `Makefile`? I would strongly prefer having them at the toplevel...if not `docs/releasing.md` needs updating.